sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,142 @@
+# February'20 release notes
+
+Release notes for commit e8f1f29
+
+## New features
+  - Added `__builtin_intel_fpga_mem` for the FPGA SYCL device. The builtin is
+    used to indicate the characteristics of the load-store unit (LSU) to be used
+    when de-referencing the pointer [1e33c01]
+  - Added support for the `intelfpga::uses_global_work_offset` attribute
+    [8bed533]
+  - Added basic xmethod implementation for `sycl::accessor::operator[]` to make
+    it callable from gdb command line [d6be8ff]
+  - Added device libraries for standard math like std::cos and `std::complex`
+    type [7abd9d5]
+
+## Improvements
+### SYCL Frontend and driver changes
+  - Added support for passing a non type template parameter to the `loop_unroll`
+    attribute [8d7a32a]
+  - Relaxed the standard layout requirement for kernel arguments. Now by default
+    they should be trivially copiable. The `sycl-std=1.2.1` driver option turns
+    standard layout requirement "on" [3adb4a5]
+  - Added diagnostic on using `__float128` type in the device code [659efdf]
+  - The `intelfpga::max_private_copies` has been renamed to
+    `intelfpga::private_copies` [97a199f]
+  - Fixed output options behavior for `-fsycl-link` on Windows [67b24d46]
+  - Prevented duplication of error diagnostics for `-fsycl` [3a0b62e]
+  - Added diagnostic when attempting to use existing FPGA static library with
+    additional device code [6431be6]
+  - Renamed `intelfpga::uses_global_work_offset` attribute to
```

Comment:
Maybe just write one point: Added support for the `intelfpga::no_global_work_offset` attribute instead of two sentences (on line 9 and on line 30)?

Reply:
Done.